README.md

```diff
@@ -131,40 +163,44 @@ We use [Cypress](https://www.cypress.io/) for our end-to-end tests. There are a
 * `ng e2e --watch` builds and serves the client but just opens Cypress for you to be able to run the tests you want without closing automatically.
   * This is the same as running `ng serve` and `npm run cy:open` (or `npx cypress open`) at the same time. If you are already running `ng serve` it will be easier to do this rather than closing it and running `ng e2e`.
 
-The main page of Cypress looks like this: 
+> When Nic ran the current version of Cypress for the first time popped up a
+> (macOS) "validation" check. This timed out and failed the first time, but when
+> he ran it again it worked fine the second time.
```

Comment:
I think this is a very unlikely one time fluke and I don't think it's necessary to mention it here. We would go crazy if we tried to mention every single one-time odd error we have had.
```suggestion
```

Reply:
Agreed. I just don't _know_ it won't happen to another student, and we do have some stuff with Macs. If it doesn't happen to anyone else, then we'll definitely want to remove it.